### PR TITLE
ESLint components that were missed

### PR DESCRIPTION
**Pull request by SeanArmstrong**

### CODE REVIEW

File: src/components/table/table-header/table-header.js [hunk 1]
```diff
@@ -96,31 +104,73 @@ class TableHeader extends React.Component {
...
+   * @return {Object}
+   */
+  get tableHeaderProps() {
+    const { ...props } = validProps(this);
 
-    this.context.onSort(this.props.name, sortOrder);
+    delete props.children;
+
+    props.className = this.tableHeaderClasses();
+    props.onClick = this.props.sortable ? this.emitSortEvent.bind(this) : '';
```
Comment (paulsturgess) [inline src/components/table/table-header/table-header.js]: I realise this is not your code, but is the `.bind` actually necessary here? `emitSortEvent` is defined using the fat arrow syntax.

Comment (SeanArmstrong) [inline src/components/table/table-header/table-header.js]: Ill remove it whilst im here. Good spot

File: src/components/modal/modal.js [hunk 2]
```diff
@@ -154,29 +161,34 @@ class Modal extends React.Component {
...
-          className="carbon-modal__background"
+          className='carbon-modal__background'
         />
       );
     }
+    return null;
   }
 
+  /* eslint-disable no-useless-return */
+
```
Comment (abramin) [inline src/components/modal/modal.js]: couldn't you do `return null` here? At the moment it's returning `undefined`

Comment (SeanArmstrong) [inline src/components/modal/modal.js]: Good point, thought it was complaining as there was a return with no content.

This makes more sense

**Pull request merged**